### PR TITLE
Phm 483/create device resource

### PR DESCRIPTION
**Pull request by malabbasi01**
https://jira.onemedical.com/browse/PHM-483

Implantable device tests, ensure all metadata fields are present according to http://hl7.org/fhir/us/core/STU5/StructureDefinition-us-core-implantable-device.html

### CODE REVIEW

**Review by alyssa-gardiner — CHANGES_REQUESTED** [review]
This looks awesome! A console log snuck in there but other than that looks great!!

**Review by AnithaPal — DISMISSED** [review]
🔥  🎉  Looks good ship it

File: src/lib/handlebars-converter/handlebars-helpers.js [hunk 1]
```diff
@@ -578,6 +627,7 @@ module.exports.external = [
                         }
                     }
                 }
+                // console.log(ret)
```
[the remark this answers is not in the document]
Comment (malabbasi01) [reply src/lib/handlebars-converter/handlebars-helpers.js]: removed

File: src/templates/cda/Resources/Device2.hbs [hunk 2]
```diff
@@ -9,17 +9,29 @@
...
+        "type": {
+            "coding": [
+                {{#each (toArray deviceEntry.participant.participantRole.playingDevice.code) as |code|}}
+                    {
+                    "code":"{{code.code}}",
+                    "display":"{{code.displayName}}",
+                    "system":{{>ValueSet/SystemReference.hbs code=code.codeSystem}},
+                    },
+                {{/each}}
+<!--                codeable concept from participant.participant.playingDevice.code-->
```
[the remark this answers is not in the document]
Comment (malabbasi01) [reply src/templates/cda/Resources/Device2.hbs]: removed

**Review by alyssa-gardiner — APPROVED** [review]

**Review by alyssa-gardiner — APPROVED** [review]

**Pull request merged**